odoo/addons/base/tests/test_res_company.py

```diff
@@ -50,3 +50,41 @@ def test_logo_check(self):
         self.assertTrue(company.uses_default_logo)
         company.partner_id.image_1920 = "R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
         self.assertFalse(company.uses_default_logo)
+
+    def create_companies(self):
+        env = self.env['res.company']
+        companies = [env.create({'name': f'company_{i}'}) for i in range(4)]
+        return env.browse([company.id for company in companies])
```

Comment:
this can be written:

```suggestion
        return self.env['res.company'].create([{'name': f'company_{i}'} for i in range(4)]
```
side note: env is a shortcut for environment, so calling "res.company" empty recordset as env variable is confusing. A name we usually use would be "Company" or "ResCompany".